src/modules/DNS.rb

```diff
@@ -479,6 +482,21 @@ def Import(settings)
 
       @nameservers = Builtins.eval(Ops.get_list(settings, "nameservers", []))
       @searchlist = Builtins.eval(Ops.get_list(settings, "searchlist", []))
+
+      # check for AY unsupported scenarios, the name servers and the search domains
+      # are written in the 2nd stage, if is disabled then it cannot work (bsc#1046198)
```

Comment:
So, we're fixing a bug which is not a bug. Somebody tried to use the module by an unsupported way and we'll take care of it. 

In my POV the issue should be closed as invalid. Instead of adding a check for one of several unsupported cases we should improve documentation for SP3 to make more obvious what is and is not supported.